main/lsp/LSPTypechecker.cc

```diff
@@ -415,16 +415,14 @@ bool LSPTypechecker::runSlowPath(LSPFileUpdates updates, WorkerPool &workers, bo
     // Note: This is important to do even if the slow path was canceled. It clears out any typechecking errors from the
     // aborted typechecking run.
     // Note 2: `gs` now holds the value of `finalGS`.
-    auto out = gs->errorQueue->drainWithQueryResponses();
+    gs->errorQueue->drainWithQueryResponses();
```

Comment:
Since we create a brand new error queue for every typecheck run, I think we can get rid of this now with no consequences.